src/app/static/src/app/components/adr/SelectDataset.vue

```diff
@@ -61,15 +93,37 @@
                             <span class="font-weight-bold">${d.organization.title}</span>
                         </div>`
                 }))
+            },
+            newDataset() {
+                const fullMetaData = this.datasets.find(d => d.id = this.newDatasetId)
+                return fullMetaData && {
+                    id: fullMetaData.id,
+                    title: fullMetaData.title,
+                    revision_id: fullMetaData.revision_id,
+                    url: `https://adr.unaids.org/${fullMetaData.type}/${fullMetaData.name}`
```

Comment:
Ok, so this is the reason the url isn't working. I've hard-coded "https://adr.unaids.org" as the base url even though when running in development the back-end is talking to "https://dev.adr.fjelltopp.org/". I did think of adding a HINT endpoint GET `adr/url` which the front-end could call to get back the configured ADR location but it seemed like a faff and I thought hard-coding the url here would suffice for now.

Reply:
(nb, if you added your production ADR key, the reason it still worked is because the API keys on their dev server are exactly the same as on their prod server!)

Reply:
Actually just realised we already have an endpoint returning the individual resource schema names, should just add the base url into that endpoint. Will add that in on a separate branch.